packages/management/featuregates/bundle/config/upstream/tanzu-featuregates-manager.yaml

```diff
@@ -25,6 +25,8 @@ spec:
         - image: featuregates-controller-manager:latest
           imagePullPolicy: IfNotPresent
           name: manager
+          args:
+            - #@ "--webhook-server-port={}".format(data.values.deployment.webhookServerPort)
```

Comment:
question: is `webhookServerPort` required in values.yaml ? If so, do we have .star script to do the validation ? One concern I have is that if the following cases break our code ?

Case 1:
```
#@data/values
---

namespace:
deployment:
  hostNetwork: true
  nodeSelector: null
  tolerations: []
```

Case 2:
```
#@data/values
---

namespace:
deployment:
  hostNetwork: true
  nodeSelector: null
  tolerations: []
  webhookServerPort: 9443c
```

Reply:
@danniel1205 added some validations to validate webhookServerPort, let me know what you think